cpp/onedal/accessor.hpp

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#pragma once
+
+#include "onedal/table_builder.hpp"
+
+namespace dal {
+
+template <typename AccessType>
+class row_accessor {
```

Comment:
We've been talking how to support read-only, write-only and read-write modes. The given interface supports only read-only and read-write modes.

Reply:
Thanks, it's a good point. Previously we decided to postpone this topic and figure it out in our prototype (there were discussions about compile-time/runtime access hints for write-only access, e.g.). This is one of the upcoming tasks for me.